mlflow/diviner.py

```diff
@@ -57,6 +64,7 @@
 _MODEL_BINARY_FILE_NAME = "model.div"
 _MODEL_TYPE_KEY = "model_type"
 _FLAVOR_KEY = "flavors"
+_SPARK_MODEL_INDICATOR = "fit_in_spark"
```

Comment:
```suggestion
_SPARK_MODEL_INDICATOR = "fit_with_spark"
```

Reply:
Updated in Diviner branch as well to be consistent (the empty file written with be `_fit_with_spark`)

Reply:
Makes sense :)